pkgs/by-name/fi/fiji/package.nix

```diff
@@ -21,7 +22,14 @@ stdenv.mkDerivation rec {
 
   dontBuild = true;
 
-  nativeBuildInputs = [ autoPatchelfHook makeWrapper copyDesktopItems unzip ];
+  nativeBuildInputs = [
+    autoPatchelfHook
+    wrapGAppsHook3
```

Comment:
Double wrapping with `makeWrapper`. Consider adding `dontWrapGApps=true`, and
```
  makeWrapper ... \
    ''${gappsWrapperArgs[@]}
```

Reply:
Thanks! I changed it as suggested.

Reply:
Does it work for you?

Reply:
Yes, it build and runs fine.